GOVERNANCE.md

```diff
@@ -0,0 +1,253 @@
+- Decisions that affect multiple teams should go through the proposal process.
+- All proposals should be discussed publicly in the appropriate GitHub issue or pull request.
+- If a member of appropriate team feels feedback from members of the same or other teams is warranted they will @mention the users or teams to request feedback.
+- Proposals may also be added to the Flux Dev weekly meetings agenda, as a good avenue for making progress on a decision <https://lists.cncf.io/g/cncf-flux-dev/calendar>.
+- If there are objections and no consensus can be found, a vote may be called by a team member.
+- When a vote is called, members of the appropriate team team will cast their yes/no vote on that github issue or pull request, and after a suitable period of time, the votes will be tallied and the outcome noted.
+- If a proposal cannot be resolved by the affected teams, the decision may be escalated to the Org team.
+
+## Voting
+
+- Most team decisions are made by lazy consensus <https://communitymgt.wikia.com/wiki/Lazy_consensus>.
+- In these cases, if no consensus can be reached, the matter may be resolved by Simple Majority Vote <https://en.wikipedia.org/wiki/Majority> by the appropriate team.
+- However there are cases that always require a Simple Majority or Supermajority Vote <https://en.wikipedia.org/wiki/Supermajority> by the appropriate team.
+  These are defined below.
+
+### Simple Majority Changes
+
+- Org team: Licensing and intellectual property changes.
+- Org team: Using Flux/CNCF money for anything CNCF deems "not cheap and easy".
+
+### Supermajority Changes
+
+- Org team: Enforcing a Code of Conduct violation.
+- Org team: Removing a member from a team for any reason other than inactivity.
+- Org team: Material changes to the Governance document.
+  - Editorial changes to governance may be made by Org team lazy consensus, unless challenged.
+    These are changes which fix spelling or grammar, update work affiliation or similar, update style or reflect an outside and obvious reality.
+    They do not change the intention or meaning of anything in this document.
+
+### Membership Changes
+
+- All team members MUST remain active contributors.
+  If they are unresponsive for > 3 months they will be automatically removed unless a supermajority of the other members of that team agrees to extend the period.
+- A member may voluntarily step down from a team by following the proposal process in the community repo.
+  These proposals are automatically accepted.
+  Once offboarding is complete and the issue is closed, they become a retired member of that team.
+- New members may be added to a team by a supermajority vote by the existing members of that team.
+- When a team has no members the Org team become responsible for it and may either archive the team or find new members.
+- When a team is archived, its members become retired members of that team.
+
+Dev team membership changes have additional considerations:
```

Comment:
Is there anything the Dev team decides that isn't scoped to a project? I reckon either make it explicitly and  _only_ per-project (and exactly the maintainers of each project), or don't treat it as a team alongside the others.

Reply:
This is out of date with current simplifications.